flink-dist/src/main/assemblies/opt.xml

```diff
@@ -95,5 +95,12 @@
 			<destName>flink-metrics-statsd-${project.version}.jar</destName>
 			<fileMode>0644</fileMode>
 		</file>
+
+		<file>
+			<source>../flink-metrics/flink-metrics-datadog/target/flink-metrics-datadog-${project.version}.jar</source>
```

Comment:
The `maven-jar-plugin` creates 2 jars. For the datadog reporter these are, right now, 

1. `flink-metrics-datadog-1.3-SNAPSHOT.jar`
2. `flink-metrics-datadog-1.3-SNAPSHOT-jav-with-dependencies.jar`

The first jar doesn't contain any dependencies, whereas the second one contains all dependencies declared in the pom.xml, as long as the aren't set to `provided`. We want the latter to be placed in `/opt`, as such the `<source>` tag must point to that jar.

Basically replace the existing `<source>` configuration with `<source>../flink-metrics/flink-metrics-datadog/target/flink-metrics-datadog-${project.version}-jar-with-dependencies.jar</source>`